gunicorn/workers/workertmp.py

```diff
@@ -35,14 +36,12 @@ def __init__(self, cfg):
             os.close(fd)
             raise
 
-        self.spinner = 0
-
     def notify(self):
-        self.spinner = (self.spinner + 1) % 2
-        os.fchmod(self._tmp.fileno(), self.spinner)
+        new_time = time.monotonic()
+        os.utime(self._tmp.name, (new_time, new_time))
```

Comment:
Gunicorn unlinks the temporary file as soon as its created, so that it doesn't exist in the directory listing but is still open for the worker. I think you need to use `fileno()` here, still, which is why I double-checked that `utime()` supports a file descriptor.